spec/features/like_feature_spec.rb

```diff
@@ -0,0 +1,25 @@
+feature 'Likes' do
+
+  before do
+    # sign_up
+    # post_a_picture
+  end
+
+  scenario 'a user can like a picture, which updates the likes count', js: true do
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)